lib/windlopen.c

```
@@ -136,7 +136,7 @@ int _sasl_get_plugin(const char *file,
     newhead = sasl_ALLOC(sizeof(lib_list_t));
     if (!newhead) return SASL_NOMEM;
 
-    if (!(library = LoadLibrary (file))) {
+    if (!(library = LoadLibraryA (file))) {
```

This also isn't required.  However if the build is performed with the unicode switch on, loading external plugins silently fails without this change, because then LoadLibrary will be expecting a unicode path and it's always passed a utf-8 string. LoadLibraryA always expects a UTF-8 string.